ansible/README_DISTRIBUTED.md

```diff
@@ -0,0 +1,110 @@
+Deploying Distributed OpenWhisk using Ansible
+=========
+
+### Distributed Deployment using Ansible
+**Caveat:** All Ansible commands are meant to be executed from the `ansible` directory.
+This is important because that's where `ansible.cfg` is located which contains generic settings that are needed for the remaining steps.
+
+By default, if you omit the `-i` parameter in ansible commands, the `local` environment will be used.
+
+In all instructions, replace `<openwhisk_home>` with the base directory of your OpenWhisk source tree. e.g. `openwhisk`
+
+Login to your bootsrapper VM. Your local machine can act as bootstrapper as well, if it can connect to other VMs.
+
+Install prereqs for ansible
+```
+sudo apt-get -y install python-pip python-setuptools python-dev libssl-dev
+```
+
+Install Ansible and cloud module packages:
+```
+sudo pip install ansible==2.0.2.0 markupsafe shade pitz positional appdirs monotonic rfc3986 jsonschema
```

Comment:
That being said, i'ld say we move the comment on running `all.sh` up here (ansible for example is already included there). And add the dependencies we need afterwards. Imho it'ld be fine to install 1-2 additional pip packages even in `all.sh` even though they are not necessarily needed for a local deployment.